app/src/main/java/fr/free/nrw/commons/mwapi/OkHttpJsonApiClient.java

```diff
@@ -328,6 +329,76 @@ public List<Place> getNearbyPlaces(final LatLng cur, final String language, fina
         throw new Exception(response.message());
     }
 
+    /**
```

Comment:
Do we still need the methods above and below this one (with the `radius` parameter)?

Reply:
I think it's required during upload to prepopulate categories and depictions based on the image coordinates where their `radius` is used.

Reply:
Sure!